src/game/objects/bridge.c

```diff
@@ -4,6 +4,17 @@
 #include "game/draw.h"
 #include "game/objects/cog.h"
 #include "global/vars.h"
+#include "src/config.h"
+
+bool isSameSector(int32_t x, int32_t y, int32_t z, ITEM_INFO *item)
```

Comment:
This function doesn't adhere to the naming conventions outlined in contributing guidelines.
ITEM_INFO * can be const.